js/src/framework.ts

```diff
@@ -194,6 +200,16 @@ export interface Evaluator<
    * and compared to this experiment. This takes precedence over `baseExperimentName` if specified.
    */
   baseExperimentId?: string;
+
+  /**
+   * Optional settings for collecting git metadata. By default, will collect all git metadata fields allowed in org-level settings.
+   */
+  gitMetadataSettings?: GitMetadataSettings;
```

Comment:
should we propagate them to the eval api as well?

Reply:
Good idea. Done.